EricGray/main.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="/Users/pcsstudent/PCS/jse-win15-6/EricGray/homework6.js"></script>
+<script src="/Users/pcsstudent/PCS/jse-win15-6/EricGray/cardsFactory.js"></script>
```

Comment:
In general, you should avoid using absolute paths for filenames in HTML files&mdash;relative paths are much more flexible and allow you to move your files around, which you will almost certainly do when going from production to live. for example, a simple `<script src="homework6.js"></script>` would work better here.